### ISSUE TITLE
Cls loss not decreasing

### ISSUE THREAD

**Issue by oliviervluijk** [zylo117/Yet-Another-EfficientDet-Pytorch#741]
Hi everyone, 
I have already checked #285 & #714, but in both issues the tutorial works with their code. The shape tutorial indeed works with lr = 1e-4, but the logo tutorial doesn't work lr = 1e-4. With the logo tutorial my cls loss is all over the place (something like the picture below).
<img width="658" alt="results_rip" src="https://user-images.githubusercontent.com/114149238/197174747-71b9e508-a660-4f68-aeca-c2355893a7ef.png">

EDIT: After 37 epochs, the loss is around ~1.1, but when running the eval.py script and look at the average precision and average recall, almost all numbers are 0.00.

Anyone knows what the problem could be?

**Comment by hoxjun** [comment]
I have same problem. #361 I changed learning rate and optimizer, but AP is still not good. 
Please help me TT
